utils/frame/benchmarking-cli/src/machine/mod.rs

```diff
@@ -184,7 +184,7 @@ impl MachineCmd {
 		);
 		// Print the final result.
 		if failed != 0 {
-			info!("The hardware fails to meet the requirements");
+			warn!("The hardware fails to meet the requirements");
```

Comment:
@ordian So I should only change this to `warn` and that is it, right? I will close the companions then since they are not needed.